python/unitytrainers/trainer_controller.py

```diff
@@ -257,16 +258,22 @@ def start_learning(self):
             else:
                 sess.run(init)
             global_step = 0  # This is only for saving the model
-            self.meta_curriculum.increment_lessons(self._get_progresses())
-            curr_info = self.env.reset(config=self.meta_curriculum.get_config(), train_mode=self.fast_simulation)
+            if self.meta_curriculum is not None:
```

Comment:
Since these five lines are used twice here, does it make sense to break them out into their own "check curriculum and reset env" function?